app/app.go

```diff
@@ -265,7 +265,7 @@ func NewWasmApp(
 	// TODO: remove amino codec dependency once Tendermint version is upgraded with
```

Comment:
Can this todo go away? Or is appCodec still amino?

(having app.cdc and appCodec was super confusing)